### PR TITLE
Add reconciliation exceptions for polygon 20231017

### PR DESCRIPTION
**Pull request by anon-r-7**
Critical update to account for accounting exception in chainlink metrics on polygon network.

### CODE REVIEW

File: models/chainlink/polygon/chainlink_polygon_schema.yml [hunk 1]
```diff
@@ -1053,3 +1053,25 @@ models:
...
+      project: chainlink
+      contributors: ["linkpool_ryan"]
+    config:
+      tags: ["chainlink", "ocr", "reward", "daily", "polygon"]
+    description: >
+      Chainlink OCR Reconcile Daily
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - blockchain
```
Comment (Hosuke) [inline models/chainlink/polygon/chainlink_polygon_schema.yml:1069]:
```suggestion
```

Comment (Hosuke) [reply models/chainlink/polygon/chainlink_polygon_schema.yml:1069]: Or we can add `blockchain` into `unique_key`.

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @anon-r-7 

[Sample chainlink_polygon_ocr_reconcile_daily](https://dune.com/queries/3116003)
[Sample chainlink_polygon_ocr_reward_daily](https://dune.com/queries/3116013)

File: models/chainlink/polygon/chainlink_polygon_ocr_reconcile_daily.sql [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      MAX(cast(evt_transfer.value as double) / 1e18) as token_value
+    FROM
+      {{ source('erc20_polygon', 'evt_Transfer') }} evt_transfer
+    LEFT JOIN
+      {{ ref('chainlink_polygon_ocr_operator_admin_meta') }} ocr_operator_admin_meta ON ocr_operator_admin_meta.admin_address = evt_transfer."from"
+    WHERE
+      evt_transfer.evt_block_time >= cast('2023-10-16' as date)
+      AND evt_transfer."to" = 0x2431d49d225C1BcCE7541deA6Da7aEf9C7AD3e23    
+      {% if is_incremental() %}
+      AND evt_transfer.evt_block_time >= date_trunc('day', now() - interval '{{incremental_interval}}' day)
```
Comment (aalan3) [inline models/chainlink/polygon/chainlink_polygon_ocr_reconcile_daily.sql:32]: can you please use the `incremental_predicate` macro here and below instead?

**Review by aalan3 — COMMENTED** [review]
Could you provide an explanation for why this is needed in the description? These tables seem to fix a specific day but at the same time they are incremental, could they be static instead?

**Pull request merged**